exporter/otlpexporter/testdata/config.yaml

```diff
@@ -14,6 +14,11 @@ retry_on_failure:
   multiplier: 1.3
   max_interval: 60s
   max_elapsed_time: 10m
+batcher:
+    enabled: true
+    flush_timeout: 200ms
+    min_size_items: 1000
+    max_size_items: 10000
```

Comment:
Indentation is a bit off compare to others.

Reply:
Thanks!